.github/workflows/publish.yml

```diff
@@ -11,22 +11,7 @@ jobs:
       contents: read
 
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v3
-
-      - name: Use Node.js 18.x
-        uses: actions/setup-node@v2.2.0
-        with:
-          node-version: '18.x'
-
-      - name: Install
-        run: yarn
-
-      - name: Format
-        run: yarn format
-
-      - name: Check formatting
-        run: git diff --exit-code
+      - uses: linz/action-typescript@v2
```

Comment:
I did not know we had this action doing all of that

Reply:
Ha yeah I made it a while back cause i got sick of copy pasting all the same logic everywhere then having to bump all the versions!